_includes/posts/maintenance-table.md

```diff
@@ -1,4 +1,5 @@
 | 22.0 | 2021-09-13 | 2022-11-24  | 2023-04-01 |
```

Comment:
I think `22.0` can be removed now?

Reply:
Going to leave it as-is for a month or two.